src/main/java/com/denizenscript/ddiscordbot/events/DiscordLeaveUserScriptEvent.java

```diff
@@ -0,0 +1,86 @@
+package com.denizenscript.ddiscordbot.events;
+
+import net.aufdemrand.denizencore.events.ScriptEvent;
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.objects.dObject;
+import net.aufdemrand.denizencore.scripts.containers.ScriptContainer;
+import net.aufdemrand.denizencore.utilities.CoreUtilities;
+import sx.blah.discord.handle.impl.events.guild.member.UserLeaveEvent;
+
+public class DiscordLeaveUserScriptEvent extends ScriptEvent {
+    public static DiscordLeaveUserScriptEvent instance;
+
+    // <--[event]
+    // @Events
+    // discord user leaves (for <bot>)
+    //
+    // @Regex ^on discord user leaves(for [^\s]+)?$

```

Comment:
bad regex - missing a space in front of `for`

Reply:
Like this? 
`@Regex ^on discord user leaves( for [^\s]+)?$`
I also assume this should also be changed on all the other meta events

Reply:
yes